lavalink/players.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, bot, guild_id):
     @property
     def position(self):
         if not self.paused:
-            diff = time() - self._position_timestamp
-            return min(self._position + diff, self.current)
+            diff = time()*1000 - self._position_timestamp
+            return min(self._position + diff, self.current.duration)
+        return min(self._position, self.current.duration)
```

Comment:
Not sure what `min` is used for. Should never exceed self.current.duration, anyway.